tests/sentry/api/test_permissions.py

```diff
@@ -5,51 +5,38 @@
     SuperuserOrStaffFeatureFlaggedPermission,
     SuperuserPermission,
 )
-from sentry.auth.staff import Staff
-from sentry.auth.superuser import Superuser
 from sentry.testutils.cases import TestCase
 from sentry.testutils.helpers import with_feature
 
 
 class PermissionsTest(TestCase):
     def setUp(self):
-        # self.user is superuser and staff user by default
-        self.request: Request = self.make_request(user=self.user)  # type: ignore
-        self.request.superuser = Superuser(self.request)
-        self.request.staff = Staff(self.request)  # type: ignore[attr-defined]
-        self.login_as(self.user)
-
-    def _activate_superuser(self):
-        self.request.superuser.uid = str(self.user.id)
-        self.request.superuser._is_active = True
-
-    def _activate_staff(self):
-        self.request.staff.uid = str(self.user.id)
-        self.request.staff._is_active = True
+        self.superuser_request: Request = self.make_request(user=self.user, is_superuser=True)  # type: ignore
+        self.staff_request: Request = self.make_request(user=self.user, is_staff=True)  # type: ignore
```

Comment:
why are we ignoring the type of this again?

Reply:
the helper returns httprequest but we expect drf request in the permission checks

Reply:
i see. could you cast `HttpRequest` into `Request`?

Reply:
I tried that but it was overriding something in the helper